agot-bg-game-server/src/common/ingame-game-state/IngameGameState.ts

```diff
@@ -391,6 +391,10 @@ export default class IngameGameState extends GameState<
             player.user.note = message.note.substring(0, NOTE_MAX_LENGTH);
         }
 
+        if (this.game.pausedForUpdate && !this.hasChildGameState(CombatGameState) && !this.entireGame.gameSettings.onlyLive) {
```

Comment:
@Longwelwind This is the most relevant line which does the magic… The rest helps to organize the pause. I extended and released`getViewOfGame()` today to show the games in combat on the games page..